packages/spiderfier/README.md

```diff
@@ -0,0 +1,60 @@
+# Marker Spider for AGM
+
+-----
+
+this package levereges the [npm-overlapping-marker-spiderfier][npm-overlapping-marker-spiderfier] to add spiderfy
+support to [AGM][agm].
+
+## Installation
+
+@agm/js-marker-spider has a peer depedency on [npm-overlapping-marker-spiderfier][npm-overlapping-marker-spiderfier]
+
+```shell
+npm install npm-overlapping-marker-spiderfier @agm/js-marker-spider --save
```

Comment:
We should use a version range here for `npm-overlapping-marker-spiderfier`  that works with our package.

Reply:
I actually made a full ts rewrite of the package that is 100% compatible but forgot to update the readme, I will update and include version info